### ISSUE TITLE
upgrade RuboCop to >= 0.59.1

### ISSUE THREAD

**Issue by tomas-nava** [department-of-veterans-affairs/caseflow#8488]
Our current version of RuboCop (`0.52.1`) has a bug that flags properly formatted `strftime` tokens like the ones in this line:

```ruby
expect(page).to have_content(legacy_task.assigned_at.strftime("%m/%d/%Y"))
```

with a [`Style/FormatStringToken` error](https://rubocop.readthedocs.io/en/latest/cops_style/#styleformatstringtoken).

This bug was [fixed at least by RuboCop `0.59.1`](https://github.com/rubocop-hq/rubocop/issues/6308). The current version is `0.62.0`. Is there any reason we shouldn't upgrade?

**Comment by mdbenjam** [comment]
No concerns here.

**Comment by tomas-nava** [comment]
This was closed by #8397
